DDF.xml

```diff
@@ -216,7 +216,7 @@ These LwM2M Object Resources MUST only be changed by a LwM2M Bootstrap-Server or
         <Owner>OMA</Owner>
         <Source>0</Source>
         <Ver>1.0</Ver>
-        <DDF>LWM2M_LOCKWIPE-v1_0_1.xml</DDF>
+        <DDF>LWM2M_LOCKWIPE-v1_0_2.xml</DDF>
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
         <TS>http://www.openmobilealliance.org/release/LWM2M_LOCKWIPE/V1_0_1-20181003-A/OMA-TS-LWM2M_LockWipe-V1_0-20180615-A.pdf</TS>
```

Comment:
LOCKWIPE TS will require updating to incorporate the changes to v1_0_1 - Did you want to update the links in this PR?